rotkehlchen/chain/ethereum/tokens.py

```diff
@@ -33,10 +36,117 @@
+    Asset('eip155:1/erc20:0x272F97b7a56a387aE942350bBC7Df5700f8a4576'),
+    Asset('eip155:1/erc20:0xc9BC48c72154ef3e5425641a3c747242112a46AF'),
+    Asset('eip155:1/erc20:0xF256CC7847E919FAc9B808cC216cAc87CCF2f47a'),
+    Asset('eip155:1/erc20:0x1982b2F5814301d4e9a8b0201555376e62F82428'),
+    # Maker tokens
+    Asset('eip155:1/erc20:0x6B175474E89094C44Da98b954EedeAC495271d0F'),
+    Asset('eip155:1/erc20:0x9f8F72aA9304c8B593d555F12eF6589cC3A579A2'),
+    Asset('eip155:1/erc20:0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2'),
+    Asset('eip155:1/erc20:0x0D8775F648430679A709E98d2b0Cb6250d2887EF'),
+    Asset('eip155:1/erc20:0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48'),
+    Asset('eip155:1/erc20:0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599'),
+    Asset('eip155:1/erc20:0x0000000000085d4780B73119b644AE5ecd22b376'),
+    Asset('eip155:1/erc20:0xE41d2489571d322189246DaFA5ebDe1F4699F498'),
+    Asset('eip155:1/erc20:0xdd974D5C2e2928deA5F71b9825b8b646686BD200'),
+    Asset('eip155:1/erc20:0x0F5D2fB29fb7d3CFeE444a200298f468908cC942'),
+    Asset('eip155:1/erc20:0xdAC17F958D2ee523a2206206994597C13D831ec7'),
+    Asset('eip155:1/erc20:0x8E870D67F660D95d5be530380D0eC0bd388289E1'),
+    Asset('eip155:1/erc20:0xc00e94Cb662C3520282E6f5717214004A7f26888'),
+    Asset('eip155:1/erc20:0xBBbbCA6A901c926F240b89EacB641d8Aec7AEafD'),
+    Asset('eip155:1/erc20:0x514910771AF9Ca656af840dff83E8264EcF986CA'),
+    Asset('eip155:1/erc20:0xba100000625a3754423978a60c9317c58a424e3D'),
+    Asset('eip155:1/erc20:0x0bc529c00C6401aEF6D220BE8C6Ea1667F6Ad93e'),
+    Asset('eip155:1/erc20:0x056Fd409E1d7A124BD7017459dFEa2F387b6d5Cd'),
+    Asset('eip155:1/erc20:0x1f9840a85d5aF5bf1D1762F925BDADdC4201F984'),
+    Asset('eip155:1/erc20:0xEB4C2781e4ebA804CE9a9803C67d0893436bB27D'),
+    Asset('eip155:1/erc20:0x7Fc66500c84A76Ad7e9c93437bFc5Ac33E2DDaE9'),
+    Asset('eip155:1/erc20:0x7D1AfA7B718fb893dB30A3aBc0Cfc608AaCfeBB0'),
+    Asset('eip155:1/erc20:0x028171bCA77440897B824Ca71D1c56caC55b68A3'),
+    Asset('eip155:1/erc20:0x06325440D014e39736583c165C2963BA99fAf14E'),
+]
+
+
 class EthereumTokens(EvmTokens):
 
-    def __init__(self, database: 'DBHandler', ethereum_inquirer: 'EthereumInquirer') -> None:
-        super().__init__(database=database, evm_inquirer=ethereum_inquirer)
+    def __init__(
+            self,
+            database: 'DBHandler',
+            ethereum_inquirer: 'EthereumInquirer',
+            proxies_mapping: dict[ChecksumEvmAddress, ChecksumEvmAddress],
+    ) -> None:
+        super().__init__(
+            database=database,
+            evm_inquirer=ethereum_inquirer,
+            proxies_mapping=proxies_mapping,
+        )
+        # Add Maker tokens
+        self.tokens_for_proxies = [
+            asset.resolve_to_evm_token()
+            # set() is needed because there are duplicates in TOKEN_COLLATERAL_TYPES (for example, WBTC)  # noqa: E501
+            for asset in set(COLLATERAL_TYPES_MAPPING.values())
+            if asset.is_evm_token()
```

Comment:
Why this if? Shouldn't they all be evm tokens except the `A_ETH`?

Wouldn't it be simpler in makerdao/constants.py to define two dicts, one being an update of the other, and to take the first dict, the one with only tokens in this loop and avoid the if?

Reply:
Alright, will do it this way